locale/en.json

```diff
@@ -195,39 +192,39 @@
     "reasonForAppeal": {
       "title": "Home Office Decision",
       "heading": "Why do you think the Home Office decision is wrong?",
-      "usefulInfo": "It may be helpful to look at the <a class=\"evidence-list__link\" href='#'>Home Office documents</a> (opens in a new window). They include your decision letter and other information about your case.",
-      "textTitle": "<span  class='govuk-body govuk-!-font-weight-bold'> Tell us in your own words</span>  - you don't have to have perfect English.",
+      "usefulInfo": "It may be helpful to look at the <a class=\"govuk-link\" target='_blank' href='/view/home-office-documents'>Home Office documents</a> (opens in a new window). They include your decision letter and other information about your case.",
```

Comment:
Don't want urls in this file

Reply:
Also have a look at the other target='_blank' some rel tag that needs adding for zap.

Reply:
Thank you just applied the above suggestions as part of the following commit